projects/hslayers/src/components/layermanager/layermanager-metadata.service.ts

```diff
@@ -197,6 +197,7 @@ export class HsLayerManagerMetadataService {
       if (layerObj == undefined) {
         return;
       }
+      //TODO: This should be removed probably to not pollute layer object. Use cachedCapabilities instead
       olLayer.setProperties(layerObj);
```

Comment:
I don't even see any reason, why we should do this, as we already check for all the necessary properties in add data services.